config/index.js

```diff
@@ -342,6 +342,14 @@ var conf = convict({
     env: 'DEVICE_UPDATES_ENABLED',
     default: true
   },
+  // A safety switch to disable device-driven notifications,
+  // in case problems with the client logic cause server overload.
+  deviceNotificationEnabled: {
+    doc: 'Are device-driven notifications enabled?',
+    format: Boolean,
+    env: 'DEVICE_NOTIFICATIONS_ENABLED',
```

Comment:
nit: for consistency, I think we should use consistent pluralization between setting name and env var, i.e. "Notification" vs "NOTIFICATIONS".  But I don't particularly care which one gets picked.